python/orca/tutorial/NCF/process_spark_dataframe.py

```diff
@@ -83,7 +83,8 @@ def string_index(df, col):
     indexer = StringIndexer(inputCol=col, outputCol=col + '_index').fit(df)
     df = indexer.transform(df)
     df = df.drop(col).withColumnRenamed(col + '_index', col)
-    df = df.withColumn(col, df[col].cast('int'))
+    # The StringIndexer output is float type.
```

Comment:
Add one more comment `# Change to 1-based index with 0 reversed for unknown features.`